### ISSUE TITLE
Feature Request (Optional Full Message)

### ISSUE THREAD

**Issue by jdrsqrd** [peterhajas/MobileNotifier#214]
I really like MobileNotifier and I think that if Apple is paying attention at all this is the model they will use when they go for it in iOS5 (assuming they will). However my only flaw with it is that sometimes I don't want to open the message or later it but I can't get the FULL message so I have to open it regardless just to see. It would be nice to have the option to see full messages, not in the pop up view, but in the list view.

**Comment by m-dierker** [comment]
Agreed. We should definitely be able to see the entire notification without leaving the app we're in.

**Comment by jdrsqrd** [comment]
Yeah like I have certain apps where I only need to see the notification to go on with my day. (CNN, Weather apps, and Even some Twitter notifications like "@insertname is following you") I assume it's being worked on but thought it wouldn't hurt to ask.
